lms/envs/test.py

```diff
@@ -175,6 +175,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': 'default.db',
```

Comment:
nit: remove this change

Reply:
@zubair-arbi The tests for middlewares in `edly-edx-app` fail without it is db marked by pytest and use this config.

Reply:
done. The tests for middleware were failing before. Looks like the error message was misleading.